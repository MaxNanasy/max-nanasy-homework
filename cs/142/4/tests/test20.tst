type t1 = array 10 of int;
short proc(short a) { return a; }
main()
{
  var a : t1;
  const c = 11;
  get(a[1]);
}

