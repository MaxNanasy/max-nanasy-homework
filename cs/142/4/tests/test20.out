***** Symbol Table Contents *****
N: int C: TYPE T: null
N: void C: TYPE T: null
N: short C: TYPE T: null
N: t1 C: TYPE T: int D: 10
N: proc C: PROCEDURE T: short
N: MAIN C: PROCEDURE T: void
